dns/run

```diff
@@ -22,17 +22,17 @@ outDir=${2:-"out"}
 nodeLocalIP=${4:-"169.254.20.10"}
 mkdir -p ${outDir}
 
-pip2 install numpy
+pip install numpy
 
 case "$1" in
   kube-dns )
-    python2 py/run_perf.py --params params/kubedns/default.yaml --out-dir ${outDir} --use-cluster-dns
+    python py/run_perf.py --params params/kubedns/default.yaml --out-dir ${outDir} --use-cluster-dns
```

Comment:
You need to change it to python3 explicitly. Otherwise, python2 will be used as python still links to python2

For consistency and clarity, please do the same for pip, i.e. use pip3 explicitly

Reply:
Yeah. actually i didn't do it on purpose, to see if the tests will fail if python3 is not specified. I will make the relevant changes.

Reply:
done.